cli/command/cli.go

```diff
@@ -50,6 +52,7 @@ type StorageOSCli struct {
 	client          *api.Client
 	hasExperimental bool
 	defaultVersion  string
+	timeout         int
```

Comment:
Why store this as an int?

It's idiomatic to store time durations as a `time.Duration` in Go, as there is no question regarding the unit of duration (seconds, milliseconds, minutes). All `time.Duration` values are stored in nanoseconds, and it has a bunch of helpful methods to convert to other bases.

```suggestion
	timeout         time.Duration
```

Reply:
done